notebooks/mock-notebooks/login/User Login Flow.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>To clarify as a guest user is the idea that they can search for datasets on this domain but cannot do anything else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Yes correct, they can only explore datasets but cannot access them. They will need to register themselves to the domain in order to do so.</p>

Reply:
<p>Let's add some detail to the syft.register prompt... like "As a guest, you can look around and see what data is on this Domain. When you're ready to begin your data science project, create an account by calling syft.register().</p><p> </p><p>Alternatively it might be nice to be able to call register on the guest account handle a well, just to spare having to type in all the url and port and stuff again.</p>

Reply:
<p>To rift off of Andrew...</p><blockquote>Logged in as GUEST USER</blockquote><blockquote>Explore! See what datasets are on this domain by using the command <code>data_scientist_node.datasets</code> When you're ready to begin your data science project you can create an account by calling <code>syft.register()</code></blockquote>

Reply:
👍🏾